torch/_meta_registrations.py

```diff
@@ -5718,7 +5717,14 @@ def meta__scaled_dot_product_efficient_attention(
 
     res = torch.empty(B, M, num_heads, Kv, dtype=query.dtype, device=query.device)
 
-    logsumexp_dim = math.ceil(M / 32) * 32 if compute_log_sumexp else 0
+    if torch.version.hip and torch.cuda.is_available():
```

Comment:
Does other attention backend have the same issue?

Reply:
I haven't noticed the same issue with Math or Flash, cc: @xinyazhang  for confirmation.

Reply:
@jataylo Math does not need this tensor, FA with CUTLASS does not round to multiple of 32:
https://github.com/pytorch/pytorch/blob/bd019c0bb485904a99fb38589444b1461ab1e486/aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.cpp#L456

In other words, rounding to multiple of 32 is specific to ME on CUDA.